src/gui/sdlmain.cpp

```diff
@@ -1862,27 +1974,31 @@ Bitu GFX_SetSize(int width,
 		    !sdl.desktop.window.adjusted_initial_size) {
 			sdl.desktop.window.adjusted_initial_size = true;
 			safe_set_window_size(desired_w, desired_h);
-			SDL_GetWindowSize(sdl.window, &windowWidth, &windowHeight);
+			SDL_GetWindowSize(sdl.window, &window_width, &window_height);
 		}
 
+		int canvas_width  = 0;
+		int canvas_height = 0;
```

Comment:
For consistency, suggest: `SDL_Rect canvas = ();`

Reply:
Done.